projects/zeromq.cmake

```diff
@@ -0,0 +1,37 @@
+########################################
+# novas 
+########################################
+xpProOption(zeromq)
+set(ZEROMQ_VERSION 4.2.2)
+set(ZEROMQ_SRC_PATH ${CMAKE_BINARY_DIR}/xpbase/Source/zeromq)
+set(ZEROMQ_DLURL https://github.com/zeromq/libzmq/archive/v${ZEROMQ_VERSION}.tar.gz)
+# 4.2.1 Hash = 820cec2860a72c3257881a394d83bfc0
```

Comment:
This is probably ok to take out now if 4.2.2 is the version that is going to be used.